container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java

```diff
@@ -117,6 +120,18 @@ public void run(Handle h, boolean firstTime) {
         }
     }
 
+    private void warmupQueries() {
+        String[] requestUris = new String[] {"warmupRequestUrl1", "warmupRequestUrl2"};
```

Comment:
This was probably not intended in this pull request ?

Reply:
Thanks - removed